packages/css/src/field.css

```diff
@@ -14,7 +14,7 @@
   /**
    * States
    */
-  &:has([aria-disabled='true'], :disabled) > * {
+  &:has(> [aria-disabled='true'], > :disabled) > * {
```

Comment:
This would will break the selector as `disabled` might be wrapped in affix :(

Reply:
@eirikbacker should be fixed now